### ISSUE TITLE
[Request] Please add a volume slider.

### ISSUE THREAD

**Issue by ZaChelbom** [int-72h/oftoast-godot#24]
The sounds that play at the beginning/end of the verify/update processes are very loud even though my system volume is set to 39 out of 100. Adding a volume slider that effects the notification & error handler would be nice as they tend to be way too loud.

**Comment by Sloofy** [comment]
If a volume slider does get added, I'd really appreciate it if it wasn't at 100% to start with. Like, current audio levels being what they are at max volume and the default setting being somewhere at half.

**Comment by ZaChelbom** [comment]
Yeah maybe start it at 0 on first startup and prompt the user to change to their desired volume. You would probably also need to add an audio test button or have the program play a sound when you change the volume.

**Comment by int-72h** [comment]
Implemented in alpha5 but not very well - needs tweaking.
